packages/form-js-editor/test/helper/index.js

```diff
@@ -176,4 +188,20 @@ export async function setEditorValue(editor, value) {
 
   // Requires 2 ticks to propagate the change to bpmn-js
   await act(() => {});
-}
\ No newline at end of file
+}
+
+export async function expectNoViolations(node, options = {}) {
```

Comment:
```suggestion
export { expectNoViolations } from '../../form-js-viewer/test/helper';
```

Should we simply re-export the same helpers here?